roles/openshift_health_checker/openshift_checks/docker_storage.py

```diff
@@ -143,7 +143,7 @@ def _get_vg_free(self, pool, task_vars):
                 "so the available storage in the VG cannot be determined.".format(pool)
             )
         vg_name = match.groups()[0].replace("--", "-")
-        vgs_cmd = "/sbin/vgs --noheadings -o vg_free --select vg_name=" + vg_name
+        vgs_cmd = "/sbin/vgs --noheadings -o vg_free --units g --select vg_name=" + vg_name
```

Comment:
Of course, better to make small changes to fix the BZ now, and defer any bigger change, if any.